docs/api/BayesianNeuralFieldMAP.md

```diff
@@ -0,0 +1,3 @@
+# BayesianNeuralFieldMAP
+
+::: bayesnf.spatiotemporal.BayesianNeuralFieldMAP
```

Comment:
do you need the `.spatiotemporal` given the `__init__.py` change?

Reply:
The fully qualified path is needed
```
ERROR   -  Could not collect 'bayesnf.BayesianNeuralFieldEstimator'
```